include/triton/core/tritonserver.h

```diff
@@ -91,7 +91,7 @@ struct TRITONSERVER_MetricFamily;
 ///   }
 ///
 #define TRITONSERVER_API_VERSION_MAJOR 1
-#define TRITONSERVER_API_VERSION_MINOR 18
+#define TRITONSERVER_API_VERSION_MINOR 19
```

Comment:
@GuanLuo @Tabrizian I notice this was bumped a few times in recent PRs (model namespacing, tracing request IDs, etc.) - do we want to bump this for each change or just group the recent API version changes once within the release?

Reply:
I think it is fine to bump it for each change. It doesn't cost anything :)